src/workers/cdc/primary.ts

```diff
@@ -740,6 +744,48 @@ const onPostRelationChange = async (
   }
 };
 
+const onMarketingCtaChange = async (
+  con: DataSource,
+  data: ChangeMessage<MarketingCta>,
+) => {
+  if (data.payload.op !== 'u') {
+    return;
+  }
```

Comment:
What if it's deletion?

Reply:
It cascades and deletes the rows in UserMarketingCta, which triggers the other change and deletes them then.

Reply:
Ah got it :)

Reply:
Actually we do need to be careful deleting it could potentially schedule a lot of cdc messages, let's say 100_000 users are in some campaign and it gets deleted, that could clog our cdc worker for quite a lot.

Reply:
I can remove the deletion button from the retool app, that way they can't trigger a delete

Reply:
I've disabled the delete button in Retool